public/docs/dart/latest/guide/displaying-data.jade

```diff
@@ -431,16 +430,14 @@
 
     p.
       The app's entry point—<code>main.dart</code>—imports
-      <code>bootstrap.dart</code>.
+      <code>angular2.dart</code> and <code>bootstrap.dart</code>.
       Both <code>show_properties.dart</code> and <code>friends_service.dart</code>
-      import <code>angular2.dart</code> instead,
-      because they use Angular APIs but aren't entry points.
-      (They don't call <code>bootstrap()</code>.)
-      See <a href="/docs/dart/latest/quickstart.html#performance">Performance,
-      the transformer, and Angular 2 libraries</a> for more information.
+      import only <code>angular2.dart</code>
+      because they aren't entry points.
```

Comment:
Ah, so I should remove the angular2.dart import from all the web/main.dart files that don't use @Component (or any other annotation). That affects every example except for the very first ones (getting started & quickstart). I blame WebStorm. It didn't mark the import as unused. :)